tests/data_validation/data_validation_generics_side_panel_component.test.ts

```diff
@@ -155,6 +155,48 @@ describe("data validation sidePanel component", () => {
     expect(errorMessageEl?.textContent).toContain("The range is invalid.");
   });
 
+  test("Can remove a valid, invalid or empty range", async () => {
+    await simulateClick(".o-dv-add");
+    await nextTick();
+    await changeCriterionType("dateIs");
+
+    setInputValueAndTrigger(".o-selection-input input", "A1:A4");
+    expect(document.querySelectorAll(".o-selection-input input")).toHaveLength(1);
+
+    // Add & remove a valid range
+    await simulateClick(".o-add-selection");
+    const range1 = document.querySelectorAll(".o-selection-input input")[1];
+    await setInputValueAndTrigger(range1, "B1:B4");
+    const remove1 = document.querySelectorAll(".o-remove-selection")[1];
+    await simulateClick(remove1);
+    expect(document.querySelectorAll(".o-selection-input input")).toHaveLength(1);
+
+    // Add & remove an invalid range
+    await simulateClick(".o-add-selection");
+    const range2 = document.querySelectorAll(".o-selection-input input")[1];
+    await setInputValueAndTrigger(range2, "B1:HOLA");
+    const remove2 = document.querySelectorAll(".o-remove-selection")[1];
+    await simulateClick(remove2);
+    expect(document.querySelectorAll(".o-selection-input input")).toHaveLength(1);
+
+    // Add & remove an empty range
+    await simulateClick(".o-add-selection");
+    const remove3 = document.querySelectorAll(".o-remove-selection")[1];
+    await simulateClick(remove3);
+    expect(document.querySelectorAll(".o-selection-input input")).toHaveLength(1);
+
+    const composer = ".o-dv-settings .o-composer";
+    await editStandaloneComposer(composer, "=DATE(2025,1,1)");
```

Comment:
```suggestion
    await editStandaloneComposer(".o-dv-settings .o-composer", "=DATE(2025,1,1)");
```
nitpick. The variable is kinda useless